samples/meeting-live-coding-interview/csharp/MeetingLiveCoding/ClientApp/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
```

Comment:
How much of this is still needed? Or can we remove the react-create-app boilerplate?